src/textures/CompressedArrayTexture.js

```diff
@@ -0,0 +1,18 @@
+import { ClampToEdgeWrapping } from '../constants.js';
+import { CompressedTexture } from './CompressedTexture.js';
+
+class CompressedArrayTexture extends CompressedTexture {
+
+	constructor( depth, mipmaps, width, height, format, type, mapping, wrapS, wrapT, magFilter, minFilter, anisotropy, encoding ) {
```

Comment:
Let's follow the order of the ArrayTexture constructor here...

https://github.com/mrdoob/three.js/blob/d081c5a3501d272d19375fab1b01fedf9df29b22/src/textures/DataArrayTexture.js#L6

I would suggest skipping the arguments after `depth` (layers) for now. It is easy to add them later, but hard to change the order once they're added, and we may need to think about the order a bit for compatibility with other texture classes.

```javascript
constructor( mipmaps, width, height, depth ) { ... }
```